import numpy as np
import pandas as pd 
import matplotlib.pyplot as plt

#===<!data!>===
Товари_shop = ["Яблука", "Банани", "Виноград", "Апельсини"]
Товари = np.array(Товари_shop)
#------------------------------------------------------------------------
Кількість_продаж_shop = [45, 12, 30, 24]
Кількість = np.array(Кількість_продаж_shop)
#------------------------------------------------------------------------
Ціна_за_1_кг_shop = [25, 40, 60, 35]
Ціна_за_1_кг = np.array(Ціна_за_1_кг_shop)
#------------------------------------------------------------------------

#===<!df!>===

df = pd.DataFrame({
    'Товар' : Товари,
    'ціна' : Ціна_за_1_кг,
    'Кількість' : Кількість
})
#------------------------------------------------------------------------

df['Загальна вартість'] = df['Кількість'] * df['ціна']
df['mean_ціна'] = df['ціна'].mean()
#===<!Output!>===

print("data")
print(df)
#------------------------------------------------------------------------



df.plot(
    x='Товар',
    y='Кількість',
    kind='bar',
    title='Продажі товарів (кг)',
    legend=False,
    color='skyblue'
)
plt.xticks(rotation=45)
plt.tight_layout()
plt.show()